icu4c/source/i18n/unicode/measfmt.h

```diff
@@ -91,7 +91,8 @@ class DateFormat;
 /**
  * <p><strong>IMPORTANT:</strong> New users are strongly encouraged to see if
  * numberformatter.h fits their use case.  Although not deprecated, this header
- * is provided for backwards compatibility only.
+ * is provided for backwards compatibility only, and has much more limited
+ * capabilities.
```

Comment:
A minor touching of unicode/*.h ...

Maybe it should be a different PR? It's here mostly because it was in response to considering unit tests for MeasureFormat, but then recognising its use is strongly discouraged.